secure_message/common/alerts.py

```diff
@@ -27,18 +27,18 @@ def send(email, reference):
                                  json=notification)
 
         if response.status_code != 201:
-            raise RasNotifyException(code=500)
+            raise RasNotifyException(code=500, survey_id=survey_id, party_id=party_id)
```

Comment:
But the notify is only called on message post. And message post swallows all exceptions in _alert_listeners( The reason being that the message post was deemed a success even if the notify failed - business choice). So raising a 500 here is misleading as it will never get back to the caller, however it WILL lead to log entries . So its possibly worth a note to say that the exception is swallowed and the caller does not get a 500 in this circumstance